content/articles/momentum-based-gradient-optimizer-and-optimization-techniques-for-gradient-descent/index.md

```diff
@@ -0,0 +1,133 @@
+   alt: Gradient Descent Based Optimization Techniques example image
+---
+
+### Introduction
+
+Alternative models can be built using Gradient Descent in machine learning systems. An objective function specifies the level of inaccuracy that a Machine Learning model will have on a given dataset during the training phase.
+
+A random set of values will be used throughout training to maintain fairness. As a result, we get closer to the ideal function value with each repetition.
+
+The quick convergence of Adaptive Optimization Algorithms, on the other hand, is causing them to gain popularity. For better convergence resilience, these methods diverge from the traditional Gradient Descent algorithm.
+
+### Table of contents
+
+- [Optimization techniques](#optimization-techniques)
+- [Momentum-based Optimization](#momentum-based-optimization)
+- [Stochastic Gradient Descent](#stochastic-gradient-descent)
+- [Adaptive Moment Estimation](#adaptive-moment-estimation)
+- [RMSProp](#rmsprop)
+- [AdaDelta](#adadelta)
+- [AdaGrad](#adagrad)
+- [Mini Batch Stochastic Gradient Descent](#mini-batch-stochastic-gradient-descent)
+- [Various other optimizers that have emerged in the last few years](#various-other-optimizers-that-have-emerged-in-the-last-few-years)
+- [Optimizer to use](#optimizer-to-use)
+
+### Optimization techniques
+For a brief while, we'll consider various optimization methods frequently employed in Deep Learning. In other words, if you came to this article hoping to learn the equation and mathematical explanation, don't bother. 
+
+This page also includes a quick overview of the many approaches that are accessible, including those provided by the Keras package.
+
+### Momentum-based Optimization
+
+Utilize an Adaptive Optimization Algorithm (AOA) that uses exponentially-weighted averaging of gradients from prior rounds to keep convergence from deviating. Most real-world applications, such as deep neural networks, rely on noisy input to train. 
+
+There must be a systematic approach to feeding the data to minimize the impact of noise during optimization. To resolve this issue, exponentially weighted averages might be used.
+
+Using an iterative process, we can approximate the trends in a P-dimensional noisy dataset by calculating the parameters as we go through the data:
+```
+On iteration p:
+    Get next  θp 
+    v_θ = β vθ + (1 - β)θp
+```
+**Vθ** is averaged across its preceding **1/(1 - β)** iterations, giving a value of **vθ**. With the help of averages, you can eliminate all the randomness and merely keep track of the overall trend. As a result of this method, the algorithm is more resistant to sample noise and can be trained more quickly.
+### How does momentum help in gradient descent
+To describe a gradient descent algorithm, the term `momentum` is used. This method works by detecting the direction of the steepest slope in its present state and then travelling in the direction of that sharpest slope to update its status.

```

Comment:
This is factually incorrect. Momentum is not used to describe the algorithm, it is additional information used to converge faster.